diff --git a/mm/slub.c b/mm/slub.c
index cfd56e5a35fb..3bc96a46be67 100644
--- a/mm/slub.c
+++ b/mm/slub.c
@@ -2724,6 +2724,10 @@ static __always_inline void *slab_alloc_node(struct kmem_cache *s,
 
 	slab_post_alloc_hook(s, gfpflags, 1, &object);
 
+	if(strstr(current->comm, "poc") || strstr(current->comm, "syz-executor"))
+	{
+		pr_err("[*] ALLOC cache : %s, %llX\n", s->name, object);
+	}
 	return object;
 }
 
@@ -3897,6 +3901,10 @@ void kfree(const void *x)
 		return;
 	}
 	slab_free(page->slab_cache, page, object, NULL, 1, _RET_IP_);
+	if(strstr(current->comm, "poc") || strstr(current->comm, "syz-executor"))
+	{
+		pr_err("[*] FREE cache : %s, %llX\n", page->slab_cache->name, x);
+	}
 }
 EXPORT_SYMBOL(kfree);
 
