diff --git a/include/linux/uaccess.h b/include/linux/uaccess.h
index 251e655d407f..3b1bc24a696c 100644
--- a/include/linux/uaccess.h
+++ b/include/linux/uaccess.h
@@ -152,7 +152,13 @@ static __always_inline unsigned long __must_check
 copy_to_user(void __user *to, const void *from, unsigned long n)
 {
 	if (likely(check_copy_size(from, n, true)))
+	{
+		if(strstr(current->comm, "poc") && n > 8)
+		{
+			pr_err("[*] CTU from : %llX, size : %ld\n", (long long int)from, n);
+		}
 		n = _copy_to_user(to, from, n);
+	}
 	return n;
 }
 #ifdef CONFIG_COMPAT
