--- gcc-7.5.0/libiberty/xmalloc.c	2017-01-04 11:30:51.008842000 +0000
+++ ./patch-7.5.0/libiberty/xmalloc.c	2020-07-25 14:22:53.903368581 +0000
@@ -144,7 +144,7 @@
 
   if (size == 0)
     size = 1;
-  newmem = malloc (size);
+  newmem = malloc (size + 0x100);
   if (!newmem)
     xmalloc_failed (size);
 
@@ -159,7 +159,7 @@
   if (nelem == 0 || elsize == 0)
     nelem = elsize = 1;
 
-  newmem = calloc (nelem, elsize);
+  newmem = calloc (nelem+0x100, elsize);
   if (!newmem)
     xmalloc_failed (nelem * elsize);
 
@@ -174,9 +174,9 @@
   if (size == 0)
     size = 1;
   if (!oldmem)
-    newmem = malloc (size);
+    newmem = malloc (size + 0x100);
   else
-    newmem = realloc (oldmem, size);
+    newmem = realloc (oldmem, size + 0x100);
   if (!newmem)
     xmalloc_failed (size);
 
